docs/user/vsphere/customization.md

```diff
@@ -12,7 +12,11 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 
 ## Machine pools
 
-There are currently no configurable vSphere-specific machine-pool properties.
+* `osDisk` (optional object):
+    * `diskSizeGB` (optional integer): The size of the disk in gigabytes (GB).
+* `numCPUs` (optional integer): The total number of virtual processor cores to assign a vm.
+* `numCoresPerSocket` (optional integer): The number of cores per socket in a vm. The number of vCPUs on the vm will be NumCPUs/NumCoresPerSocket (default is 1).
+* `memoryMiB` (optional integer): The size of a VM's memory in megabytes.
```

Comment:
Why MiB ?

Reply:
Aligning to the machineapi
https://github.com/openshift/machine-api-operator/blob/master/pkg/apis/vsphereprovider/v1alpha1/vsphereproviderconfig_types.go#L49

Reply:
Terraform says MB and the machine-api-operator says MiB. I'm going to assume this is a stylistic issue of using MiB/MB interchangeably rather than a technical issue of one using base10 and one using base2. I will update this code to use MB and GB to be consistent with existing code and I think red hat docs. @kalexand-rh, thoughts?

Reply:
Looking at other comments, I think we need to balance the user-facing consistency with consistency with the machine-api-operator. I think there is value in keeping the variables to have the same names as those in the machine-api-operator. Therefore I would propose we use GB/MB user-facing but keep MiB/GIB for code variable names. This would result in https://github.com/openshift/installer/pull/2938/files/31256a8a3bac5318b0b9844f2fd892fba2afa12f#diff-5614f874d3fff51453d911222e551672R14 becoming:
```
	MemoryMiB int64 `json:"memoryMB"`
```